.github/workflows/nix.yml

```diff
@@ -0,0 +1,17 @@
+name: "Nix Env Build"
+on:
+  pull_request:
+  push:
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: cachix/install-nix-action@v31
+      with:
+        nix_path: nixpkgs=channel:nixos-unstable
```

Comment:
sounds like asking for random CI issues. Can we use a stable channel?

Reply:
Good catch.

Reply:
Switched to the latest relased version (24.11) which seems to be the way to get a stable.